train.py

```diff
@@ -1099,8 +1102,8 @@ def prepare_and_start_instructor(opt):
 
     # statedicts
     if opt.state_dict is not None:
-        if os.path.isfile(opt.state_dict):
-            # if exists, nothing to do
+        if os.path.isfile(opt.state_dict) or opt.state_dict == "pretrained":
```

Comment:
But it basically boils down to which state_dict is used, doesn't it? Thats why I used it. The other parameters are not influenced by it.
Adding a new parameter (e.g. `--pretrained`) would then mean to ignore `--state_dict`, correct?
So the command line invocation would look like this:
```sh
python infer.py --pretrained
```
which would be the same as currently
```sh
python infer.py --state_dict pretrained
```
Also, what I don't understand: Why does `own_model_name` only make sense jointly with grutsc? Aren't others allowed here: https://github.com/fhamborg/NewsMTSC/blob/e95c7b5943095e80253ad072d9a6049609e72dad/train.py#L74-L100

Reply:
> I think a better solution is to introduce a new py file / method that contains the download / setup procedure and invoke that from setup.py.

I don't think that's a good solution. IMO a pypi-package shouldn't automatically download models. E.g. if someone only wants to use the library for training their own model, it shouldn't take that much disk space.
However we could do it in a similar way to spacy, so that we require another command invocation after installation to automatically download the model.